egs/mgb5/s5/local/prepare_data.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Copyright 2019 QCRI (Author: Ahmed Ali)
+# Apache 2.0
+
+set -e -o pipefail
+
+
+###
+# The script assumes you have downloaded to the MGB-5 corpus: https://arabicspeech.org/mgb5
+# DB/{dev.tar.gz,train.tar.gz}
+###
```

Comment:
can you please check for existence of the files and print an informative error message if they don't exist?

Reply:
done.